src/components/Compendia/Download.js

```diff
@@ -135,131 +136,146 @@ export const Download = ({ type }) => {
-      >
-        Choose Organism
-      </Box>
-      <Box style={{ position: 'relative' }}>
-        <Box
-          style={{
-            position: 'absolute',
-            right: '10px',
-            top: '10px',
-            zIndex: 1
-          }}
-        >
-          {!loading && <Icon name="ChevronDown" size="xsmall" />}
-        </Box>
-        {loading ? (
-          <Spinner />
-        ) : (
-          <DropDown style={{ position: 'relative' }}>
-            <SearchBox
-              padding="16px 32px"
-              placeholder="Search for an organism"
-              size="small"
-              reverse={false}
-              responsive
-              value={userInput}
-              onChange={(e) => handleChange(e.target.value)}
-              onFocus={handleFocus}
-            />
-            {showOptions && filteredOptions.length > 0 && (
-              <Box
-                animation={{ type: 'zoomIn', duration: 50 }}
-                background="white"
-                border={{ color: 'brand', size: 'medium' }}
-                margin={{ top: 'xlarge' }}
-                height={{ max: '200px' }}
-                width="100%"
-                style={{
-                  overflowY: 'scroll',
-                  position: 'absolute',
-                  zIndex: 1
-                }}
-              >
-                <List flexDirection="column">
-                  {filteredOptions.map((option) => (
-                    <ListItem
-                      key={option.primary_organism_name}
-                      label={formatString(option.primary_organism_name)}
-                      selectedOrganism={
-                        selectedOrganism
-                          ? formatString(selectedOrganism.primary_organism_name)
-                          : null
-                      }
-                      onClick={() => handleClick(option)}
-                    />
-                  ))}
-                </List>
-              </Box>
-            )}
-          </DropDown>
-        )}
-      </Box>
-      {type === 'rnaSeq' && (
-        <Box margin={{ top: setResponsive('small', 'medium') }}>
-          <InlineMessage label="Data is not normalized or aggregated." />
-        </Box>
-      )}
-      {selectedOrganism && selectedOrganism.organism_names.length > 1 && (
-        <Box margin={{ top: 'large' }}>
-          <InlineMessage
-            label={
-              <Text margin={{ left: 'small' }} style={{ display: 'block' }}>
-                Also contains small number of samples from other organisms from
-                the same species.{' '}
-                <Anchor
-                  href={links.refinebio_docs_collapsing_by_genus}
-                  label="View"
-                  rel="noopener noreferrer"
+      {hasError ? (
+        <Paragraph>
+          Download unavailable at this time. Please check again soon!
+        </Paragraph>
+      ) : (
```

Comment:
I would probably just escape hatch this before this JSX is returned to save you an indentation of nearly the entire component but this is ok too. Probably doesn't matter long term because this component is pretty long and should probably get broken up in the future.